<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>welcomeWindow</class>
 <widget class="QMainWindow" name="welcomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>welcomeWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="difficultyBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Begginner</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Easy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Medium</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hard</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Expert</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="hintsBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>67</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="newGameButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Start game</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>546</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
